NGitLab.Mock/Config/GitLabIssue.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NGitLab.Mock.Config
+{
+    /// <summary>
+    /// Describe an issue in a GitLab project
+    /// </summary>
+    public class GitLabIssue : GitLabObject<GitLabProject>
+    {
+        /// <summary>
+        /// Title (required)
+        /// </summary>
+        public string Title { get; set; }
+
+        public string Description { get; set; }
+
+        /// <summary>
+        /// Labels names
+        /// </summary>
+        public IList<string> Labels { get; } = new List<string>();
+
+        /// <summary>
+        /// Author username (required)
+        /// </summary>
+        public string Author { get; set; }
+
+        /// <summary>
+        /// Assignee username
+        /// </summary>
+        public string Assignee { get; set; }
+
+        /// <summary>
+        /// Milestone title
+        /// </summary>
+        public string Milestone { get; set; }
+
+        public DateTime CreatedAt { get; set; } = DateTime.Now;
```

Comment:
```suggestion
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
```

Reply:
BTW, I don't know if you need to set them, or just let the GitLabServer set the values when building the server.